assets/app.js

```diff
@@ -1221,6 +1233,12 @@ new Vue({
                             product.active = true
                             product.updated = Date.now()
 
+                            const re = /^CF-ST*/g
+                            if (((product['modelCode'] || '').match(re) || []).length && product['rejected_materials'])
+                            {
+                                product['rejected_materials'] = false;
```

Comment:
not sure what you do here but `startsWith()` seems to be an easier approach to figure out if its a Sticker Code